tests/test-utils/makeRequestProcessor.ts

```diff
@@ -42,6 +48,17 @@ export default function makeRequestProcessor(name: string, userOpts?: MakeFuncti
             return next('route');
          } else if (opts.passesErrorToNext) {
             return next(err);
+         } else if (opts.returnsRejectedPromise) {
+            return new Promise((_resolve, reject) => setTimeout(() => { reject(`Rejection from "${name}"`); }, 4));
```

Comment:
There is still an arrow function with an implicit return here, and below: `=> setTimeout(() =>`